client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-expanded-content/boost-site-performance.tsx

```diff
@@ -44,7 +48,7 @@ export default function BoostSitePerformance( {
 		'Your Overall Score is a summary of your website performance across both mobile and desktop devices.'
 	);
 
-	const isEnabled = !! ( hasBoost || overallScore );
+	const isEnabled = !! ( hasBoost || overallScore ) || hasPendingScore;
```

Comment:
Non-blocking: it looks like maybe the parentheses here aren't necessary, except for possibly a TypeScript complaint. An alternative might look something like this --

```suggestion
	const isEnabled = Boolean( hasBoost || overallScore || hasPendingScore );
```

Or, since the only non-Boolean value is `overallScore` --

```suggestion
	const isEnabled = hasBoost || Boolean( overallScore ) || hasPendingScore;
```

Reply:
Agree!